app/assets/javascripts/oxalis/model/helpers/nml_helpers.js

```diff
@@ -2,14 +2,34 @@
 
 import _ from "lodash";
 import { getPosition, getRotation } from "oxalis/model/accessors/flycam_accessor";
+import messages from "messages";
+import Saxophone from "@scalableminds/saxophone";
+import Store from "oxalis/store";
+import Date from "libs/date";
+import { getNodeToEdgesMap } from "oxalis/model/accessors/skeletontracing_accessor";
 import type {
   OxalisState,
   SkeletonTracingType,
   NodeMapType,
   EdgeType,
   TreeType,
+  TreeMapType,
+  TemporaryMutableTreeType,
+  TemporaryMutableTreeMapType,
 } from "oxalis/store";
 
+// NML Defaults
+const DEFAULT_TIME = 0;
+const DEFAULT_COLOR = [1, 0, 0];
+const DEFAULT_VIEWPORT = 0;
+const DEFAULT_RESOLUTION = 0;
+const DEFAULT_BITDEPTH = 0;
```

Comment:
maybe `8`?

Reply:
I am not sure about that. If we leave this at 0 we will at least know, that we could not parse the value from the NML. Maybe it would be event better to use `Option`

Reply:
What do you mean by "it would be better to use `Option`"?

Reply:
The only time we need these default values is, when we try to parse an NML files, that does not have these values. Instead of setting them to 0 (to indicate we did not parse them from the original NML) or set them to any arbitrary value (such as 8, which we did not parse either), it might make more sense to change the type op these fields from `Int` to `Option[Int]` and set them to `None` (these are backend-terms, the frontend equivalent is probably `Maybe`s). Then, on serialization, we could omit these values if they are not set and produce the same NML, that we got as input.

Reply:
It may even be better (depending on the NML spec) to expect these values to exist and fail parsing, if they don't